### ISSUE TITLE
Plugin: Could not load file or assembly System.Diagnostics.Debug

### ISSUE THREAD

**Issue by marv-th** [Splamy/TS3AudioBot#793]
Releated to the issue #787 
I try some things and add the Nuget Package System.Diagnostics.Debug but that do not reproduce the error. Visual Studio says no error and i am using the newest develop 34. And i include the newest ts3audiobot.dll and tslib.dll to the project.
It is very important to get some good help.. 
Thanks

Log: 

```
2020-05-15 16:10:13.5568|ERROR||Plugin.StartInternal Plugin 'NowPlaying (BotPlugin)' failed to load: Could not load file or assembly 'System.Diagnostics.Debug, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.
.
System.IO.FileNotFoundException: Could not load file or assembly 'System.Diagnostics.Debug, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.

File name: 'System.Diagnostics.Debug, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at System.ModuleHandle.ResolveTypeHandleInternal(RuntimeModule module, Int32 typeToken, RuntimeTypeHandle[] typeInstantiationContext, RuntimeTypeHandle[] methodInstantiationContext)
   at System.Reflection.RuntimeModule.ResolveType(Int32 metadataToken, Type[] genericTypeArguments, Type[] genericMethodArguments)
   at System.Reflection.CustomAttribute.FilterCustomAttributeRecord(MetadataToken caCtorToken, MetadataImport& scope, RuntimeModule decoratedModule, MetadataToken decoratedToken, RuntimeType attributeFilterType, Boolean mustBeInheritable, ListBuilder`1& derivedAttributes, RuntimeType& attributeType, IRuntimeMethodInfo& ctor, Boolean& ctorHasParameters, Boolean& isVarArg)
   at System.Reflection.CustomAttribute.AddCustomAttributes(ListBuilder`1& attributes, RuntimeModule decoratedModule, Int32 decoratedMetadataToken, RuntimeType attributeFilterType, Boolean mustBeInheritable, ListBuilder`1 derivedAttributes)
   at System.Reflection.CustomAttribute.GetCustomAttributes(RuntimeModule decoratedModule, Int32 decoratedMetadataToken, Int32 pcaCount, RuntimeType attributeFilterType)
   at System.Reflection.CustomAttribute.GetCustomAttributes(RuntimeMethodInfo method, RuntimeType caType, Boolean inherit)
   at System.Reflection.RuntimeMethodInfo.GetCustomAttributes(Type attributeType, Boolean inherit)
   at System.Attribute.GetCustomAttributes(MemberInfo element, Type type, Boolean inherit)
   at System.Attribute.GetCustomAttribute(MemberInfo element, Type attributeType, Boolean inherit)
   at System.Reflection.CustomAttributeExtensions.GetCustomAttribute[T](MemberInfo element)
   at TS3AudioBot.CommandSystem.CommandManager.<>c__DisplayClass23_0.<<GetCommandMethods>g__GetCommandMethodsIterator|0>d.MoveNext()
   at TS3AudioBot.CommandSystem.CommandManager.GetBotCommands(IEnumerable`1 methods)+MoveNext()
   at System.Collections.Generic.LargeArrayBuilder`1.AddRange(IEnumerable`1 items)
   at System.Collections.Generic.EnumerableHelpers.ToArray[T](IEnumerable`1 source)
   at TS3AudioBot.Plugins.PluginCommandBag..ctor(Object obj, Type t)
   at TS3AudioBot.Plugins.Plugin.CreatePluginObjects(IInjector injector, Type type, Boolean isStatic)
   at TS3AudioBot.Plugins.Plugin.StartInternal(Bot bot)

```

**Comment by Splamy** [comment]
Hey, could you describe what you changed, or if it works now? It would be helpful for others

**Comment by marv-th** [comment]
Some functions was declared as async function cause in the develop version before the current develop version i wasn't able to fetch image from url and set as avatar without async declaration.
I deleted the asynch declaration and then the error was fixed. 
Also the GetOwn function wasn't exist anymore and must renamed to book.Self(). and the function to change the bot description was changed too. VS says no error for that.
